domains/abhnv.json

```
@@ -0,0 +1,20 @@
+{
+  "owner": {
+    "username": "Abhinavv-007",
+    "email": ""
+  },
+  "record": {
+    "A": [
+      "185.199.108.153",
+      "185.199.109.153",
+      "185.199.110.153",
+      "185.199.111.153"
+    ],
+    "AAAA": [
+      "2606:50c0:8000::153",
+      "2606:50c0:8001::153",
+      "2606:50c0:8002::153",
+      "2606:50c0:8003::153"
+    ]
```

we only accept CNAME for Github Page
```suggestion
    "CNAME": "abhinavv-007.github.io"
```